source/common/json/config_schemas.cc

```diff
@@ -1305,7 +1305,7 @@ const std::string Json::Schema::CLUSTER_SCHEMA(R"EOF(
       },
       "type" : {
         "type" : "string",
-        "enum" : ["static", "strict_dns", "logical_dns", "sds"]
+        "enum" : ["static", "strict_dns", "logical_dns", "sds", "original_dst"]
```

Comment:
Can we add docs on these new options in https://github.com/lyft/envoy/blob/master/docs/configuration/cluster_manager/cluster.rst etc.